bundle/deploy/metadata/upload.go

```diff
@@ -0,0 +1,36 @@
+package metadata
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/libs/filer"
+)
+
+var MetadataFileName = "deploy-metadata.json"
```

Comment:
Just `metadata.json` is good enough, IMO.

Reply:
done